bitnami/matomo/Chart.yaml

```
@@ -39,4 +39,4 @@ maintainers:
 name: matomo
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/matomo
-version: 7.1.2
+version: 7.1.3
```

```suggestion
version: 7.2.0
```

As you are removing parameters from the values, I'd use a new minor version

---

I see your point. However, we have been using patch version in other similar PRs. Let's keep this.